pkg/api/l2geth.go

```diff
@@ -0,0 +1,105 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/tokamak-network/tokamak-titond-backend/pkg/kubernetes"
+	"github.com/tokamak-network/tokamak-titond-backend/pkg/model"
+)
+
+type l2gethConfig struct {
+	data map[string]string
+}
+
+func (t *TitondAPI) CreateL2Geth(data *model.Component /*TODO : get params for config, namespace*/) *model.Component {
+	// TODO : deal with DB
+	// t.db.CreateComponent()
+
+	/*
+		This is currently hardcoding, but
+		will be replaced by taking a value as a parameter and generating it
+	*/
+	namespace := "default"
+	config := &l2gethConfig{
+		data: map[string]string{},
+	}
+
+	config.data["ETH1_CONFIRMATION_DEPTH"] = "1"
+	config.data["GASPRICE"] = "100"
+
+	go t.createL2Geth(namespace, config)
+
+	return data
+}
+
+func (t *TitondAPI) createL2Geth(namespace string, config *l2gethConfig) {
```

Comment:
Correct me if i am wrong, I think we can only create l2geth instance one time. It maybe ok for PoC.

Reply:
I think we can make many l2geth in on time. When there are many requests in on time, gin will make goroutine for each request internally.
Please let me know if there is my missed point.

Reply:
Oh, you are correct!
we have to set the state dump file and data transport-layer endpoint for each config! I missed it.
Let me explain about the details in next our meeting. I think we can keep this code now.
Thank you!